protobuf/Makefile

```diff
@@ -1,28 +1,37 @@
 # Automatically fetch .proto files
 
+PERL=perl
```

Comment:
Did we use perl anywhere?

Reply:
oops; no... I was using it to edit the scalapb stuff (I never learned to use sed directly).

Fixed in dba8557